project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.0.0
+sbt.version=1.0.4
```

Comment:
sbt 1.0.4 is unable to build sbt 2.13.0-M1, so could you exclude it from Travis CI yml?

Reply:
Right, that's the issue. I'll drop the commit that updates sbt.version.